API/metadata/publishers_metadata.py

```diff
@@ -6,6 +6,8 @@
 from core.connectors.publishers.gcs_publisher import GCSPublisher
 from core.connectors.publishers.kafka_publisher import KafkaPublisher
 from core.connectors.publishers.s3_publisher import S3Publisher
+from core.connectors.publishers.firehose_publisher import FirehosePublisher
```

Comment:
@adithyapathipaka , would be easy if we can separate the pull request of Firehose and wrangler